cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        db_conn = None
+        max_attempts = 5
+        attempts = 0
+
+        while not db_conn and attempts < max_attempts:
+            try:
+                connections["default"].ensure_connection()
+                db_conn = connections["default"]
+            except OperationalError:
+                attempts += 1
+                self.stdout.write(
+                    f"Database unavailable, waiting 1 second... "
+                    f"(Attempt {attempts}/{max_attempts})"
+                )
+                time.sleep(1)
+
+        if db_conn:
+            self.stdout.write(self.style.SUCCESS(
+                "Database connection established!"
+            ))
+        else:
+            self.stdout.write(self.style.ERROR(
+                "Database not available after max attempts."
+            ))
```

Comment:
It would be helpful to throw an exception if the database is not available after the maximum number of attempts. This would allow the calling code to handle this situation appropriately, such as by shutting down the application or alerting the user. Currently, your code simply writes a message to the standard output, which might not be noticed in a production environment.